docs/alerts/index.md

```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
 Heartbeat | yes | no | no
 
 You can manage Alert rules interactively via the UI, or you can [manage alerts via the API](../api).
+
+## Creating an Alert Rule
+
+There are multiple methods to create an alert rule. One approach is to access any App report, hover over a chart, and click on the small alert icon that appears at the top. This action will prompt a 'Create Alert Rule' dialog, automatically populating properties such as filtering and grouping based on the settings applied within that specific chart.
+
+Alternatively, you can create an alert rule by navigating to any App report and clicking on the alert icon in the actions menu at the top of the report.
+
+Additionally, you have the option to access the Alerts page through the left menu panel and select '+ New Alert Rule' from the dropdown menu located at the top right of the screen.
```

Comment:
Looking just at the diff, so not sure if the image below shows all 3 ways of creating an AR.  Aha, I see now, it's an animated GIF and it looks like it captures them all.